L1Trigger/TrackerTFP/interface/KalmanFilterFormats.h

```diff
@@ -0,0 +1,117 @@
+#ifndef L1Trigger_TrackerTFP_KalmanFilterFormats_h
+#define L1Trigger_TrackerTFP_KalmanFilterFormats_h
+
```

Comment:
As always, classes should have a comment at top explaining what they are for. Also, I'm unclear why the KF has its own formats class, whereas all the other modules are catered for by DataFormats.h?